### DISCUSSION TITLE
IOTA's weg

### DISCUSSION THREAD

**Discussion by JTGBonheur** [iotaledger/firefly#8109] [Help & Support]
Seit dem vorletzten UPdate sind meine IOTA's bis auf 2,5 weg. Kann ich die wieder herstellen oder wo sind die hin ?

**Comment by begonaalvarezd** [comment]
You can run the balance finder from settings > advanced to search for missing funds. You can check "search on current wallet only" to speed up the process for the selected wallet you are in. Run the balance finder until your missing funds appear
